src/main/scripts/org/reaktivity/specification/mqtt/reject.publish.message.with.multiple.topic.aliases/client.rpt

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
Probably don't need the work `message` in this scenario, since we are rejecting the `publish` packet, just like we rejected the `connect` packet in another scenario.